pkg/metrics/policyfiltermetrics/policyfiltermetrics.go

```diff
@@ -4,6 +4,8 @@
 package policyfiltermetrics
 
 import (
+	"golang.org/x/exp/maps"
```

Comment:
As a note (that we will forget) [maps.Values](https://pkg.go.dev/maps@master#Values) will be moved into the standard library for 1.23.